### DISCUSSION TITLE
Code Review

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#39147] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

What are the criteria for approving a code?

**Comment by DSDmark** [comment, marked as answer]
I think it's depending upon the repo to repo if you meet the condition (if is specified in README).

If you want the way to approve code then here is :- https://linearb.io/blog/github-approve-pull-request/

**Comment by look** [comment]
@iLakshya can you clarify what you mean by approving code in your question?

**Reply by iLakshya** [reply]
I'll reframe. What I mean is that when we review a code then we get to choose from three options. One is approved another is changes requested and another is just comment.
